### ISSUE TITLE
Use JSON serialization without schema information

### ISSUE THREAD

**Issue by gabriele-ricci-kyklos** [confluentinc/confluent-kafka-dotnet#1583]
Description
===========
How can I set up JSON serialization without using a schema? I actually don't have a schema to use.

How to reproduce
================
You cannot instantiate the class `JsonSerializer<T>` without providing a json schema in its argument `ISchemaRegistryClient`

Checklist
=========

Please provide the following information:
 - [ ] A complete (i.e. we can run it), minimal program demonstrating the problem. No need to supply a project file.
 - [x] Confluent.Kafka nuget version. -> version 1.6.3
 - [ ] Apache Kafka version.
 - [ ] Client configuration.
 - [ x] Operating system. -> Windows 10
 - [ ] Provide logs (with "debug" : "..." as necessary in configuration).
 - [ ] Provide broker log excerpts.
 - [ ] Critical issue.

**Comment by mhowlett** [comment]
just use standard methods to convert to and from a string and either just use the string serializer (which is implicit, you don't need to specify it), or build your own (really simple) JSON serdes. if you're not concerned with schemas, you don't need schema registry.

**Comment by gabriele-ricci-kyklos** [comment]
You mean that I can simply produce a `Message<Null, MyCustomType>` without specifying a serializer?

**Comment by mhowlett** [comment]
no, `Message<Null, String>`. `Message<Null, MyCustomType>` will require a custom serializer, but it will be very simple to make.

**Comment by gabriele-ricci-kyklos** [comment]
Can you provide a quick example?

**Comment by douglasg14b** [comment]
Is the answer here that no, Kafka Dotnet does **not** support JSON serialization/deserialization without schema information and we must serialize and deserialize our objects to and from strings ourselves?
